Exam Labs/2018-01-17 Stairs/lab/lab.conf

```diff
@@ -0,0 +1,28 @@
+as1r1[0]="A"
```

Comment:
Please, add at the start of the lab.conf the metadata (you can see an example other labs) and feel free to put your name as author :)